### PR TITLE
update example introduction

### PR DESCRIPTION
**Pull request by koenpunt**
While looking up the definition for `Style/ClassVars` I noticed that the introduction to the example was not very well written. I must say I'm not an native English speaker either, so if there's anything that can be improved, just let me know. 

NB. It should be verified that the meaning of the text is still the same, but I think it is.

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added tests.
* [x] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [ ] Run `rake default` or `rake parallel`. It executes all tests and RuboCop for itself, and generates the documentation.

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by bbatsov** [comment]
You forgot to run the rake task that updates the manual from the source code.

**Comment by bbatsov** [comment]
@koenpunt ping :-)

File: manual/cops_style.md [hunk 1]
```diff
@@ -611,10 +611,11 @@ This cop checks for uses of class variables. Offenses
 are signaled only on assignment to class variables to
 reduce the number of offenses that would be reported.
 
-Setting value for class variable need to take care.
-If some class has been inherited by other classes, setting value
-for class variable affected children classes.
-So using class instance variable is better in almost case.
+You have to be careful when setting a value for a class
+variable; if a class has been inherited, changing the
```
Comment (pocke) [inline manual/cops_style.md]: This line contains  an trailing space, I'm not sure why this line contains it. Can you retry `bundle exec rake generate_cops_documentation`? In my environment the trailing space is removed by the rake task.

Comment (bbatsov) [reply manual/cops_style.md]: @koenpunt Ping :-)

**Pull request closed without merging**